ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml

```diff
@@ -181,29 +181,22 @@ Popup {
         anchors.topMargin: Style.current.bigPadding
         anchors.bottom: parent.bottom
 
-        TabBar {
+        StatusTabBar {
             id: walletTabBar
             width: parent.width
             anchors.top: parent.top
-            height: assetBtn.height
-            background: Rectangle {
-                color: Style.current.transparent
-                border.width: 0
-            }
 
             StatusTabButton {
                 id: assetBtn
                 //% "Assets"
-                btnText: qsTrId("wallet-assets")
-                anchors.top: parent.top
+                width: implicitWidth
+                text: qsTrId("wallet-assets")
```

Comment:
Note that we work to move away from using translation IDs. Let's refactor everytime we have a chance by adding qsTr("<User facing text here>")
Also, the `//% "Assets"` comment was related to the next line. Therefore, move this string in the `qsTr` instead.
```suggestion
                text: qsTr("Assets")
```